server/app/uk/gov/ons/addressIndex/server/controllers/PartialAddressController.scala

```diff
@@ -58,19 +58,19 @@ class PartialAddressController @Inject()(val controllerComponents: ControllerCom
     val limval = limit.getOrElse(defLimit.toString)
     val offval = offset.getOrElse(defOffset.toString)
 
-    val filterString = classificationFilter.getOrElse("").replaceAll("\\s+", "")
+    val filterString = classificationfilter.getOrElse("").replaceAll("\\s+", "")
     val endpointType = "partial"
 
     val fall = fallback.flatMap(x => Try(x.toBoolean).toOption).getOrElse(true)
     val hist = historical.flatMap(x => Try(x.toBoolean).toOption).getOrElse(true)
     val verb = verbose.flatMap(x => Try(x.toBoolean).toOption).getOrElse(false)
 
     val epochVal = epoch.getOrElse("")
-    val fromsourceVal = fromsource.getOrElse("all")
+    val fromsourceVal = {if (fromsource.getOrElse("all").isEmpty) "all" else fromsource.getOrElse("all")}
```

Comment:
This can be shortened using Scala to fromsource.filter(!_.isEmpty).getOrElse("all"). The braces are not needed.